pkg/webhook/default_server/cluster/validating/cluster_validation.go

```diff
@@ -33,16 +33,18 @@ func checkValues(c *scyllav1alpha1.Cluster) (allowed bool, msg string) {
 				return false, fmt.Sprintf("when using cpuset, you must use whole cpu cores, but rack %s has %dm", rack.Name, cores)
 			}
 
-			// Requests == Limits or only Limits must be set for QOS class guaranteed
```

Comment:
It implies that we'd have Guaranteed QoS only if CpuSet is enabled.
Isn't it that we should always use Guaranteed?

Reply:
It depends. Do we want it to be possible to run another config or not? I don't know if it's too strict or not we should hear from the field more on this. @gnumoreno what do you think? Always enforce QoS Guaranteed?

Reply:
The question is more if any user ever will run without cpuset? What are the consequences of not applying cpuset but going for Guaranteed QoS? I think none right? In case we get Guaranteed but don't give these cores to scylla it should still be fine.